articles/building-an-appointment-scheduling-telegram-bot/index.md

```diff
@@ -0,0 +1,514 @@
+Have you ever wanted to build a telegram bot that would allow you to schedule or plan your appointments? If your answer is yes, then this article is just what you need. You would also be utilizing a serverless database system called FaunaDB to build our system which will make your work easier. Now that you know what this article entails, let's get to the building part.
+
+Not so many people know about Fauna and how amazing it can be when managing databases. Therefore, I will be doing a little introduction to the Fauna serverless database system.
+
+### What is Fauna?
+
+Fauna is a client-side serverless document database that uses GraphQL and the Fauna Query Language (FQL) to support various data types and relational databases in a serverless API. You can learn more about Fauna in their official documentation [here](https://docs.fauna.com/fauna/current/).
+
```

Comment:
Delete "You can learn more about Fauna in their official documentation" and add that link to the first word "[Fauna](https://docs.fauna.com/fauna/current/)".